account_financial_report/report/templates/trial_balance.xml

```diff
@@ -36,43 +36,94 @@
             <t t-call="account_financial_report.report_trial_balance_filters" />
             <div class="act_as_table list_table" style="margin-top: 10px;" />
             <!-- Display account lines -->
+            <t t-set="aml_domain_extra" t-value="[]" />
```

Comment:
Instead of having to add this extra domain everywhere, can't we add the extra bits in the already present `domain` variable?

Reply:
I have changed to add `aml_domain_extra` only where it is really needed.

Just for clarification, you want to then add all this code:
```
<t
    t-set="aml_domain_extra"
    t-if="grouped_item['id'] > 0"
    t-value="[('analytic_account_id', '=', grouped_item['id'])]"
/>
<t
    t-set="aml_domain_extra"
    t-else=""
    t-value="[('analytic_account_id', '=', False)]"
/>
```

in the **7 places** where `aml_domain_extra` is currently added (in this case I have set global variable precisely to reduce code and do the same thing that `aml_domain_common` does for example).